doc/install.rst

```diff
@@ -28,21 +28,26 @@ Ubuntu/Mint
 .. index:: Ubuntu Linux
 .. index:: Mint
 
-We maintain a `Ubuntu Repository <https://gerbera.jfrog.io/>`__.
+We maintain a `Ubuntu Repository <https://pkg.gerbera.io/>`__.
 
-To install the latest tagged release (>=1.8.0):
+To install the latest tagged release (>=2.1.0):
 
 .. code-block:: sh
 
-    curl -fsSL https://gerbera.jfrog.io/artifactory/api/gpg/key/public | sudo apt-key add -
-    sudo apt-add-repository https://gerbera.jfrog.io/artifactory/debian
+    wget -O - https://pkg.gerbera.io/public.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/gerbera-keyring.gpg >/dev/null
```

Comment:
Don't know if this is "the usual thing" but still getting stdout on the binary gpg key here... it does however create it correctly in the target file. Logging might be a little messy if this is the norm.